packages/react-core/src/components/Card/Card.tsx

```diff
@@ -36,15 +36,28 @@ export interface CardProps extends React.HTMLProps<HTMLElement>, OUIAProps {
   isPlain?: boolean;
   /** Flag indicating if a card is expanded. Modifies the card to be expandable. */
   isExpanded?: boolean;
+  /** Flag indicating that the card should render a hidden input for a11y reasons */
+  hasHiddenInput?: boolean;
+  /** Aria label to apply to the hidden input if one is rendered */
+  hiddenInputAriaLabel?: string;
+  /** Callback that executes when the hidden input is changed */
+  onHiddenInputChange?: (labelledBy: string, event: React.FormEvent<HTMLInputElement>) => void;
```

Comment:
Only other nit at this point might be the prop names/descriptions. For now maybe tweaking the `hasHiddenInput` prop description to, "Flag indicating that the card should render a hidden input to make it selectable" ? Any further change may depend on further implementation of selectable cards/may be better for a breaking change release.

Reply:
I'm also not in love with the prop names at all, if you have any suggestions for better ones please let them fly.

Reply:
Also I really like the idea of a potential card container component, and I think that could improve a11y a *lot* while being easy for consumers to implement.

Reply:
It's a bit tougher since there's already the `isSelectable` and `isSelectableRaised` props, which if anything maybe those could be used to determine whether the hidden input is rendered with some tweaking of those prop descriptions. If those are strictly meant to determine the styling only, though, maybe `selectableInput` or `hasSelectableInput`?

`selectableInputAriaLabel` and `onSelectableInputChange` *might* also work.

Reply:
I do think we should have `isSelectable` automatically render the hidden input, but I don't think we can make that happen until the next breaking change. Changing `hiddenInput` -> `selectableInput` sounds good to me though!